identity/identity.go

```diff
@@ -86,6 +92,10 @@ func (i Identity) TableName() string {
 	return "identities"
 }
 
+func (i *Identity) InjectTraitsUrl(host string) {
+	i.TraitsSchemaURL = urlx.MustJoin(host, "schemas", i.TraitsSchemaID.String())
```

Comment:
Make this a const or move this function (partially) to `schema` package, zb: `schema.SchemaURL(id uuid.UUID)`